ironfish-cli/src/commands/reset.ts

```diff
@@ -38,11 +43,18 @@ export default class Reset extends IronfishCommand {
       HOST_FILE_NAME,
     )
 
+    let networkIdMessage = ''
+    if (flags.networkId != null) {
+      const existingId = this.sdk.internal.get('networkId')
+      networkIdMessage = `\n\nThe network ID will be changed from ${existingId} to the new value of ${flags.networkId}`
```

Comment:
nit - do we still want to print this message if `existingId === flags.networkId`?

Reply:
I don't have evidence to support one option being better -- I think skipping the message would be a bit unclear as to whether a change will happen or not, but I'm not sure if it's more helpful to have a message that says `Your network ID will stay unchanged as ${existingId}`. I'm okay with it either way and changing it if we get user feedback.

Reply:
Good call - that makes it clear in all scenarios. Change incoming